google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/TableAdminRequests.java

```diff
@@ -0,0 +1,234 @@
+     */
+    private CreateTable(String tableId) {
+      createTableRequest.setTableId(tableId);
+    }
+
+    /**
+     * Configures table with the specified granularity
+     *
+     * @param granularity
+     */
+    public CreateTable withGranularity(TimestampGranularity granularity) {
+      Preconditions.checkNotNull(granularity);
+      tableRequest.setGranularity(granularity);
+      return this;
+    }
+
+    /**
+     * Adds a new columnFamily to the configuration
+     *
+     * @param familyId
+     */
+    public CreateTable addColumnFamily(String familyId) {
+      Preconditions.checkNotNull(familyId);
+      tableRequest.putColumnFamilies(familyId, ColumnFamily.newBuilder().build());
+      return this;
+    }
+
+    /**
+     * Adds a new columnFamily with {@link GCRule} to the configuration
+     *
+     * @param familyId
+     * @param gcRule
+     */
+    public CreateTable addColumnFamily(String familyId, GCRule gcRule) {
+      Preconditions.checkNotNull(familyId);
+      tableRequest.putColumnFamilies(
+          familyId, ColumnFamily.newBuilder().setGcRule(gcRule.toProto()).build());
+      return this;
+    }
+
+    /**
+     * Adds split at the specified key to the configuration
+     *
+     * @param key
+     */
+    public CreateTable addSplit(ByteString key) {
+      Preconditions.checkNotNull(key);
+      createTableRequest.addInitialSplits(Split.newBuilder().setKey(key).build());
+      return this;
+    }
+
+    /**
+     * Adds the specified number of uniform splits to the configuration
+     *
+     * @param key
+     * @param numSplits
+    public CreateTable addUniformSplits(int numSplits) {
+      // TODO: add implementation
+      throw new UnsupportedOperationException();
+    }
+    */
+    
+    @InternalApi
+    public CreateTableRequest toProto(InstanceName instanceName) {
+      Preconditions.checkNotNull(instanceName);
+      return createTableRequest
+          .setParent(instanceName.toString())
+          .setTable(tableRequest.build())
+          .build();
+    }
+  }
+
+  /**
+   * Fluent wrapper for {@link ModifyColumnFamiliesRequest}
+   *
+   * <p> Allows for the following ColumnFamily modifications:
+   * <ul> 
+   * <li> create family, optionally with {@link GCRule}
+   * <li> update existing family {@link GCRule}
+   * <li> drop an existing family
+   * </ul>
+   */
+  public static final class ModifyFamilies {
+    private final ModifyColumnFamiliesRequest.Builder modFamilyRequest =
+        ModifyColumnFamiliesRequest.newBuilder();
+    private final String tableId;
+
+    /**
+     * Configures the tableId to execute the modifications
+     *
+     * @param tableId
+     */
+    private ModifyFamilies(String tableId) {
+      Preconditions.checkNotNull(tableId);
+      this.tableId = tableId;
+    }
+
+    /**
+     * Configures the name of the new ColumnFamily to be created
+     *
+     * @param familyId
+     * @return
+     */
+    public ModifyFamilies create(String familyId) {
```

Comment:
s/create/add
This will be make consistent with CreateTable and avoid confusion with static factory create method used in other model classes